packages/form-builder/src/blocks/fields/field.js

```diff
@@ -1,6 +1,73 @@
 import {Icon} from '@wordpress/icons';
 import {__} from "@wordpress/i18n";
 import settings from "./settings";
+import {useFieldNames} from "../../hooks";
+import {InspectorAdvancedControls} from "@wordpress/block-editor";
+import {ExternalLink, PanelRow, TextControl} from "@wordpress/components";
+
+const Edit = (props) => {
+    const ParentEdit = settings.edit
```

Comment:
Can you explain what's up with the `ParentEdit` thing?  

As a side note, we always want to avoid creating/instantiating variables, components _inside_ a component to avoid side effects and re-renders.  In this case you would want to define this before `const Edit`

Reply:
Resolved c072dee6